pkg/fqdn/cache.go

```diff
@@ -817,12 +839,17 @@ func (zombies *DNSZombieMappings) Upsert(expiryTime time.Time, addr netip.Addr,
 }
 
 // isConnectionAlive returns true if 'zombie' is considered alive.
-// Zombie is considered dead if both of these conditions apply:
+// Zombie is considered dead if all of these conditions apply:
 // 1. CT GC has run after the DNS Expiry time and grace period (lastCTGCUpdate > DeletePendingAt + GracePeriod), and
 // 2. The CG GC run did not mark the Zombie alive (lastCTGCUpdate > AliveAt)
```

Comment:
Not your change by `CG GC` should be `CT GC`, right?

Reply:
indeed indeed, good catch.